### ISSUE TITLE
elm-git.json uses ssl instead of https

### ISSUE THREAD

**Issue by gyuri-lajos** [fission-suite/drive#69]
# Summary: do not use ssl to get elm-stuff from git

## Problem

installing dependencies fails with
```bash
devDependencies:
+ elm-git-install 0.1.3
Cloning into 'elm-stuff/gitdeps/github.com/malinoff/toasty.git'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

### Impact

It is a public repo so no requirement to use ssl

## Solution

Just clone with https instead of ssl

change this in elm-git.json 
"git@github.com:malinoff/toasty.git"
to 
"https://github.com/malinoff/toasty.git"

**Comment by bmann** [comment]
@gyuri-lajos just like we use keys for fission accounts, we think using keys to interact with git are best practice. In part, because it just works with both public and private repos. 

Do you have any difficulties with using keys for git?

**Comment by gyuri-lajos** [comment]
@bmann 
I do not use github normally, I work on gitlab, being attracted there by it being opensource, 
can be selfhosted at any time,
offering private repos and promoting trunk based development.
But I see the point of putting stuff on github and working there in the future for better visibility.
I overlooked that. I should have started moving things over here a long time ago.
I sorted out SSH access.
However,
If anybody takes the url https://github.com/fission-suite/drive.git to give fission-suite/drive a whirl, will run into this problem if they do not have SSH access set up. Looking at the discussion on Stack overflow, the point is made: if the entry point is accessible via https, and so is the project that it uses accessible, it is reasonable to expect that the installing dependencies should work without requiring SSH access.

![image](https://user-images.githubusercontent.com/1477865/92310298-50ab3680-efad-11ea-89a8-353a990f1c62.png)

**Comment by bmann** [comment]
Yep, understand your point. This is not about Gitlab vs Github, but rather SSH / key based usage of git.

I will tell you that I ran into the same issue on a new machine :) This is also I've been told standard for the Elm community.

I think we will promote using keys for git, as we use them for Fission accounts as well. I'll make sure to create a guide for this and add it to READMEs by default.

Closing for now!
